src/client/store/data/slice.ts

```diff
@@ -83,61 +90,68 @@ export const dataSlice = createSlice({
     ) => {
       // Delete reference from state for deleted ODP
       const { countryIso, year, cycleName, assessmentName } = payload
-      const odpReference = state[assessmentName][cycleName].tableData[countryIso]?.originalDataPointValue?.[year]
-      if (odpReference) delete state[assessmentName][cycleName].tableData[countryIso]?.originalDataPointValue?.[year]
+      const odpReference =
+        // @ts-ignore
+        state.tableData[assessmentName][cycleName][countryIso]?.[TableNames.originalDataPointValue]?.[year]
+      if (odpReference) {
+        // @ts-ignore
+        delete state.tableData[assessmentName][cycleName][countryIso][TableNames.originalDataPointValue][year]
+      }
```

Comment:
same util method comment to extract data, plus avoid ts-ignore ?

Reply:
@ts-ignore:
![image](https://github.com/openforis/fra-platform/assets/5508251/9a71d5f6-bcc5-42ff-8482-b667ff3476ab)

Not sure about this, thoughts?